src/__tests__/collections/collections.test.js

```diff
@@ -27,6 +27,8 @@
 import fs from 'fs';
 import {flowResultTest} from '../../test-util';
 
+jest.setTimeout(10000);
```

Comment:
don't know why some tests timeout, increasing value solves it

Reply:
Can we at least add a TODO or additional task to investigate why this is needed?